package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "PracticeHTMLCSS",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "https://github.com/quocbin37/Practichtmlcss.git",
+  "author": "Pham Nhu Quoc Trieu <phamnhuquoctrieu307@gmail.com>",
+  "license": "MIT",
+  "scripts": {
+    "start": "parcel src/index.html",
+    "build": "parcel build src/index.html",
+    "posthtml": "posthtml src/index.html -o dist/index.html"
```

Comment:
for what ?

Reply:
i use it for convinient when run this practice